temporalcli/commands.workflow_exec_test.go

```diff
@@ -489,6 +489,21 @@ func (s *SharedServerSuite) TestWorkflow_Execute_EnvConfig() {
 	)
 	s.NoError(res.Err)
 	s.ContainsOnSameLine(res.Stdout.String(), "Result", `"env-conf-input"`)
+
+	// And we can specify `env` with a property
+	s.NoError(os.Setenv("TEMPORAL_ENV", "myenv"))
+	defer os.Unsetenv("TEMPORAL_ENV")
```

Comment:
You should be able to replace `LookupEnv` in the CLI options instead of using actual environment variables that affect everything globally. See `TestWorkflow_Execute_EnvVars` in `commands.workflow_exec_test.go` for an example (I saw we unfortunately have others not doing this in tests that should).

Reply:
I saw that but I though using a system property "for real" may be a better test than faking it... Are there any side-effects of using a real system property in the test?

Reply:
You can damage potential parallel tests in the future. It's mocked out at the highest level, so you're not getting much more using the real env var I don't think.